src/components/App.cy-spec.js

```diff
@@ -0,0 +1,23 @@
+// compare to App.spec.js
```

Comment:
compare to https://github.com/cypress-io/cypress-example-todomvc-redux/blob/master/src/components/App.spec.js